### PR TITLE
WT-10758 Move many-collection-test results into wiredtigers performance dashboard

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -4019,9 +4016,10 @@ tasks:
             pip3 install lorem pymongo==3.12.2 "pymongo[srv]==3.12.2"
             mongod_path=$(find ../../mongo/build -executable -type f -path \*/bin/mongod)
             ./run_many_coll.sh $mongod_path mongodb.log config/many-collection-testing many-collection clean-and-populate
-            # upload the results
-            res_dir=`find ./ -type d -name "many-collection-artifacts" -print`
-            ./upload-results-atlas.py wt-perf-tests many-collection-test ${branch_name} $res_dir/results/results.json
+      - func: "upload-perf-test-stats"
+        vars:
+          stats_dir: ./mongo-tests/largescale
```
Comment (Siddhartha8899) [inline test/evergreen.yml:4021]: Need to change this path.

File: test/evergreen.yml [hunk 2]
```diff
@@ -750,10 +750,17 @@ functions:
       params:
         file: ./wiredtiger/cmake_build/${test_path}.json
 
+  "convert-to-atlas-evergreen-format":
+    - command: shell.exec
+      params:
+        script: |
+          set -o errexit
+          set -o verbose
+          ${python_binary} wiredtiger/bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py -i ${input_file} -n ${test_name} -o ${output_path}
```
[the remark this answers is not in the document]
Comment (Siddhartha8899) [reply test/evergreen.yml:759]: I agree, we should only use specified python version

File: bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (Siddhartha8899) [reply bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py:2]: Not required, but don't mind having it there as well.

**Review by etienneptl — APPROVED** [review]

File: bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        "Test Name": test_name,
+        "metrics": [
+            {"name": key, "value": value}
+            for key, value in input_data.items()
+        ],
+        "config": {}
+    }
+    return output_data
+
+
```
Comment (sulabhM) [inline bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py:61]: Two new lines after every function seem excessive. We usually only put one.

Comment (Siddhartha8899) [reply bench/perf_run_py/perf_json_converter_for_atlas_evergreen.py:61]: This is a python convention. We follow it in Wiredtiger.

**Review by sulabhM — APPROVED** [review]
I have one comment about extra newlines that we don't need. Other than that, I only had a glance and this change LGTM.

**Pull request merged**